altar/src/render/brisk.rs

```diff
@@ -205,16 +246,20 @@ impl Sprite {
     }
 }
 
+/// Contains all of the information making up a Spritesheet. This type is never explicitly constructed, it
```

Comment:
calling `new_spritesheet` *is* explicitly constructing this; would be better to phrase as "constructed using `Spritesheets::new_spritesheet`".